deploy-service/common/src/main/java/com/pinterest/deployservice/handler/CommonHandler.java

```diff
@@ -14,29 +14,47 @@
  * limitations under the License.
  */
 package com.pinterest.deployservice.handler;
+import java.sql.Connection;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.pinterest.deployservice.ServiceContext;
-import com.pinterest.deployservice.bean.*;
```

Comment:
Yeah, it's a valid concern. However at the same time our codebase is old and some parts are not really good quality (even with formating and styling), we would encourage engineers to improve the code quality continuously when they touch some areas instead of leaving the code in bad state for long time. No need to wait for separate changes to have auto-suggested imports from VSCode. However, for more serious refactoring, totally agree it should be separate PRs. In my last company, we often do the reformating or styling change after the original code is reviewed. However, with Github flow, it requires an additional accept after accepting so I include them all here. Probably, an alternative way is to have it in 2 commits. I'm not familiar with Github so wanted to see what would be the best practice here.

Reply:
I agree, and I think it's just a judgment call based on how badly you have to avoid reverting the feature and the cost of reverting.  We have had many unexpected problems in the Teletraan codebase, so I am very cautious with my changes. Ultimately, it depends on the need to separate a future rollback or hotfix.

The additional accept is a SOX requirement, basically we must have a peer review in the final state.

Reply:
let's play on the safe side this time then :) I will revert those imports.